cli/pkg/web/assets.go

```diff
@@ -0,0 +1,49 @@
+package web
+
+import (
+	"embed"
+	"io/fs"
+	"net/http"
+	"os"
+
+	"github.com/pkg/errors"
```

Comment:
Don't use this external library, just use the standard library's `errors` package